commitlint.config.js

```diff
@@ -23,6 +23,10 @@ module.exports = {
         "mps-model-adapters",
         "mps-model-server",
         "mps-model-server-plugin",
+        "mps-sync-plugin",
+        "mps-sync-plugin-lib",
+        "sync-plugin-lib",
+        "mps-sync-lib",
```

Comment:
These are probably in here because of the linting of the oldest commits. We should remove them before merging. To do that we can override the linter check for this PR.

Reply:
> To do that we can override the linter check for this PR.

Can you help me with that, please?

Reply:
The alternative would be to reword all commits that use the old commit scopes and then force push, but that's quite a few.